packages/eslint-plugin/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## Unreleased -->
+## Unreleased
+
+### Changed
+
+- Removed `lines-around-comment` rule in `eslint-plugin`. [[#319](https://github.com/Shopify/web-configs/pull/319)]
```

Comment:
```suggestion
- Disabled `lines-around-comment` rule, let prettier handle formatting of comments. [[#319](https://github.com/Shopify/web-configs/pull/319)]
```

"in `eslint-plugin" isn't really needed as this is the CHANGELOG for the eslint-plugin package